doc/source/placement-group.rst

```diff
@@ -252,6 +252,36 @@ Note that you can anytime remove the placement group to clean up resources.
 
   ray.shutdown()
 
+Placement Group Lifetimes
+---------------
+
+.. tabs::
+  .. group-tab:: Python
+
+    Separately, placement group lifetimes can be decoupled from the job and its creator,
```

Comment:
Can you actually mention both lifetime? 1. detached. 2. Non-detached (explain this will be cleaned when the job is done and if it is created from a detached actor, it is cleaned when the detached actors are gone). I think it is not explained above.

Reply:
yeah, It's indeed, thanks for your reminding~